docs/release-notes/experiment-trial-logs-json.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Breaking Changes**
```

Comment:
This isn't breaking, since it is an additive, off-by-default, CLI flag.

Reply:
What would you recommend I call it? I was going off prior release notes for CLI --json changes

Reply:
new feature?

Reply:
i would call it 'improvements' (see https://github.com/determined-ai/determined/blame/master/docs/release-notes/launcher-327.rst#L3)